test/cli.test.js

```diff
@@ -5,22 +5,23 @@ const file = global.coverage ? "/../src/cli.js" : "/../dist/ncc/cli.js";
 
 jest.setTimeout(20000);
 
-for (const cliTest of cliTests) {
-  it(`should execute "ncc ${(cliTest.args || []).join(" ")}"`, async () => {
-    const ps = fork(join(__dirname, file), cliTest.args || [], {
+describe('cli', () => {
+  it.each(cliTests)('should execute ncc $args', async ({ args, env, expect: e, timeout }) => {
```

Comment:
Anything change here or is this just formatting?

Reply:
Formatting to use jest's built in it.each, which provides better ergonomics / outputs.

Reply:
I just wasn't sure if this could cause issues with the failing tests but seems unrelated

Reply:
I believe this shouldn't affect test emission. Also actual failure is `unit`, `cli.test` seems passing fine.

Reply:
This part seems concerning:

```
    - module.exports = eval("require")("../../../../../../../.././module");
    + module.exports = eval("require")("../../../../../../../../.././module");
```

https://github.com/vercel/ncc/runs/7774142054?check_suite_focus=true#step:9:366

That seems like its incorrect

Reply:
Yes, more problem is I couldn't see this failure on local machine.

Reply:
What about if you run `ncc cache clean`?

Reply:
I think I tried, but will give few more attempt.

Reply:
Just for the update purpose - I tried a couple like cache clean and some others, but so far no luck. I'll try to reattempt when I have some time.

Reply:
@kwonoj I can reproduce locally (macOS + Node.js 16.17.0) by checking out your branch and running

```
yarn install
yarn build
yarn test
```

<img width="696" alt="image" src="https://user-images.githubusercontent.com/229881/186439430-be19e6d3-d32a-4be2-af35-cc9ab6820ae4.png">

Reply:
I tried various, but really stuck with test result inconsistences between local to CI / other machines. It is possible my machine have some weird setup cauing problems, but without figuring it out I can't reliably say PR is ready to go.

Reply:
Do you have docker locally? You could try cloning this branch in a docker container to see if that reproduces.

Reply:
I don't think I have a commercial license to install docker on the work machines. I'll think about a few, however I'm very curious why this only (yet) occurs on my machine. Wiping out whole repo & reclone even didn't make changes.